File: JAVA/Exercicio012.java
package main;

import javax.swing.JOptionPane;

public class Exercicio012 {

	public static void main(String[] args) {
		int b = Integer.parseInt(JOptionPane.showInputDialog("Informe a base do triangulo"));
		int h = Integer.parseInt(JOptionPane.showInputDialog("Informe a altura do triangulo"));
 
		JOptionPane.showMessageDialog(null, "A área do triângulo é " + b*h/2);
	}

}

File: PHP/Exercicio012.php
<!DOCTYPE html>
<html>

<body>
 <h4>Area do triângulo (b*h/2):</h4>
 <form method="post" action="Exercicio012.php">
    <legend>Informe a base(b)</legend>
    <input type="text" name="b" id="b">
    <legend>Informe a altura(h)</legend>
    <input type="text" name="h" id="h">
    
    <button>Verificar</button>
 </form>
<?php
  if(isset($_POST['b']) && 
     isset($_POST['h'])){
      $b = $_POST['b'];
      $h = $_POST['h'];
      echo "<strong>A area do triangulo e: " . $b * $h / 2 . "</strong>";
     }
?>
</body>
</html>